src/kernel.cr

```diff
@@ -89,6 +89,13 @@ ARGV = Array.new(ARGC_UNSAFE - 1) { |i| String.new(ARGV_UNSAFE[1 + i]) }
 # ```
 ARGF = IO::ARGF.new(ARGV, STDIN)
 
+# The newline constant
+EOL = {% if flag?(:win32) %}
```

Comment:
Given this is a windows specific thing, wouldn't the `:windows` flag make more sense here? Based on https://crystal-lang.org/reference/1.7/syntax_and_semantics/compile_time_flags.html that would make it tied to the OS versus the ABI.

Reply:
Switched to checking `flag?(:windows)`. 65ee75962a7e09035af5b7525fe3a8d966610df1